source/content/guides/environment-configuration/02-read-environment-config.md

```diff
@@ -168,9 +168,9 @@ array(63) {
 
 It is not possible to set environment variables on Pantheon. However, there are three common solutions you can use instead.
 
-### Terminus Secrets Plugin
+### Terminus Secrets Manager Plugin
 
-You can use the [Terminus Secrets Plugin](https://github.com/pantheon-systems/terminus-secrets-plugin) to write the secrets to a JSON file in the private file system. Your PHP will look similar to the code example below. This example will help you get started, however, you must modify the third line for the key you want to configure. You can also modify the `secrets.json` file name, although we recommend you provide the file with a name you will recognize for secrets management.
+You can use the [Terminus Secrets Manager Plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin) to write the secrets to a JSON file in the private file system. Your PHP will look similar to the code example below. This example will help you get started, however, you must modify the third line for the key you want to configure. You can also modify the `secrets.json` file name, although we recommend you provide the file with a name you will recognize for secrets management.
```

Comment:
The sentence here describes how the old plugin works, and the example below uses the old plugin. We could update to use pantheon_get_secret(), and show how to set a secret with the Secret Manager plugin.

Reply:
https://github.com/pantheon-systems/documentation/pull/8975/commits/69025abd6c2e812c5a2a38ab4ddc237e26cc3a7a